boards/nordic/nrf54l20pdk/doc/index.rst

```diff
@@ -0,0 +1,88 @@
+.. _nrf54l20pdk_nrf54l20:
+
+nRF54L20 PDK
+############
+
+Overview
+********
+
+.. note::
+
+   All software for the nRF54L20 SoC is experimental and hardware availability
+   is restricted to the participants in the limited sampling program.
+
+The nRF54L20 Preview Development Kit hardware provides
+support for the Nordic Semiconductor nRF54L20 Arm Cortex-M33 CPU and
+the following devices:
+
+* CLOCK
+* RRAM
+* :abbr:`GPIO (General Purpose Input Output)`
+* :abbr:`NVIC (Nested Vectored Interrupt Controller)`
+* :abbr:`UARTE (Universal asynchronous receiver-transmitter)`
```

Comment:
Shouldn't this list be a bit broader, more like the one for the nRF54L15 PDK? At least in `nrf54l20_cpuapp_common.dtsi` more stuff is enabled (`adc`, `grtc`).